Firestore/core/src/core/bound.cc

```diff
@@ -82,14 +81,15 @@ bool Bound::SortsBeforeDocument(const OrderByList& order_by,
 std::string Bound::CanonicalId() const {
   std::string result = before_ ? "b:" : "a:";
   for (pb_size_t i = 0; i < position_->values_count; ++i) {
-    result.append(CanonicalId(position_->values[i]));
+    result.append(model::CanonicalId(position_->values[i]));
```

Comment:
FYI this fix from the last PR broke the build because there is a name conflict between model::CanonicalId and CanonicalId